src/Microsoft.Performance.SDK/Processing/ColumnBuilding/IModalColumnBuilder.cs

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.Performance.SDK.Processing.ColumnBuilding;
+
+/// <summary>
+///     A builder for configuring modes on a column that has been
+///     configured as having mutually exclusive modes.
+/// </summary>
+public interface IModalColumnBuilder
+    : IColumnBuilder
+{
+    /// <summary>
+    ///     Adds a mode to the column.
+    /// </summary>
+    /// <param name="modeIdentifier">
+    ///     The <see cref="ColumnVariantIdentifier"/> for the mode. The
+    ///     <see cref="ColumnVariantIdentifier.Name"/> represents the name of
+    ///     this mode.
```

Comment:
nit: I don't think this second sentence adds much. Why call out the name specifically?